scripts/apps/desks/controllers/DeskConfigController.ts

```diff
@@ -61,4 +73,70 @@ export function DeskConfigController($scope, $controller, notify, desks, WizardH
     $scope.getDeskUsers = function(desk) {
         return desks.deskMembers[desk._id];
     };
+
+    const closeModel = function() {
+        $scope.modalActive = false;
+        $scope.step.current = null;
+        $scope.desk.edit = null;
+    };
+
+    const calculateDiff = function() {
```

Comment:
Do not use field names when generating a diff. There's a library I use to generate diffs, you can check the example here https://github.com/superdesk/superdesk-client-core/blob/master/scripts/core/helpers/CrudManager.tsx#L99